ArticleTemplates/assets/scss/garnett-pillar/_colour.scss

```diff
@@ -77,41 +132,23 @@
         .element__inner {
             background-color: inherit;
         }
-    }
 
-    // Circle avatar colours
-    .avatar {
-        background-color: $p-inverted;
-    }
-
-    // Pull quotes
-    .prose .element-pullquote blockquote {
-        &:before,
-        &:after,
-        footer {
-            color: $p-kicker;
+        .meta__published {
+            color: color(shade-13);
         }
-    }
-
-    .alerts:active,
-    .alerts.following {
-        background-color: $p-kicker;
-        color: color(shade-7);
-        border-color: transparent;
-    }
 
-    .alerts,
-    .alerts.following:active {
-        border-color: rgba($p-kicker, 0.5);
-        color: $p-kicker;
-    }
-
-    .comment-count a {
-        color: $p-kicker;
+        @include pillar-reversible($p-inverted, $p-kicker, $p-feature-headline, $p-soft, $p-media-background);
```

Comment:
Is the call to `pillar-reversible` in the right place? It's a lot of code so I expected it to come up one level higher than this…